docs/4-frameworks/03-Vue.md

```diff
@@ -2,6 +2,79 @@
 
 In this tutorial you will learn how to add UI5 Web Components to your application. You can add UI5 Web Components both to new Vue.js applications and to already existing ones.
 
+## Setting up a Vite and Vue.js project with UI5 Web Components
+<br>
+
+### Step 1. Install Vue CLI.
+
+```bash
+npm install -g @vue/cli
```

Comment:
maybe the vue CLI was needed in the previous setup, now with vite this step is not necessary, please remove it.